Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -626,6 +760,51 @@ def parse_custom_whitelist(
         }
         return parsed_data
 
+    def parse_url_access_rule_group(
+        self, access_rule_group: dict[str, Any]
+    ) -> dict[str, Any]:
```

Comment:
Please add docsting describing inputs, outputs and general description about the method.